site/content/workloads-101/07-more-op-templates.md

```diff
@@ -208,9 +208,9 @@ sequence:
 ```yaml
 statements:
   - first statement body
-  - name: statement3
-    stmt: third statement body
   - second: second statement body
+  - name: third
```

Comment:
Are you removing this because it is focused more on previously supported [and currently supported] formats?

Reply:
For the further edits in this file, it makes me wonder if the uniform specification is aligned to these examples, which were mostly created before the unifor format was established. Maybe this means we should do some cross-checking and make sure that there is coverage both ways.

Reply:
I see that these changes are coherent after looking at the whole file.